examples/webgl_loader_sea3d.html

```diff
@@ -72,7 +72,7 @@
 
 				autoPlay : true, // Auto play animations
 				container : scene, // Container to add models
-				parser : THREE.SEA3D.DEFAULT, // THREE.SEA3D.BUFFER to THREE.GeometryBuffer
+				parser : THREE.SEA3D.AUTO, // Auto choose THREE.BufferGeometry and THREE.Geometry
```

Comment:
I would suggest not producing `THREE.Geometry` at all (thus removing all the code for it). And let the user convert on their code if needed.